flybywiresim/a32nx
# fix: removed unusable fuel from refueling

Fixes #4569

## Summary of Changes

Changes made in the fuel flow  caused the defueling via the efb to glitch out and rise to infinity.

Thanks to @Maximilian-Reuter @donstim for pointing out the fact that this was already fixed by https://github.com/flybywiresim/a32nx/pull/4391 but intentionally reverted after the merge of the experimental branch for compatibility with the updated fuel flow

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions

1. Spawn in and open the efb fuel page
2. Choose a lower amount than the current fuel and start defueling. It must not rise
3. Check that refueling works as before


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
